File: SistemaTrakingAplicacionMovil/app/src/main/java/com/example/admin/pruebatracking/Mensajes/MsgDBRespuesta.java
package com.example.admin.pruebatracking.Mensajes;

import com.example.admin.pruebatracking.Entidades.Cuenta;
import com.example.admin.pruebatracking.Entidades.DBEntidad;
import com.example.admin.pruebatracking.Entidades.Grupo;
import com.example.admin.pruebatracking.Entidades.Historial;

import java.util.ArrayList;

/**
 * Created by Admin on 1/11/2016.
 */
public class MsgDBRespuesta extends Mensaje {

    private String CodigoPeticion;
    private ArrayList<Cuenta> ReturnCuenta = new ArrayList<Cuenta>();
    private ArrayList<Grupo> ReturnGrupo = new ArrayList<Grupo>();
    private ArrayList<Historial> ReturnHistorial = new ArrayList<Historial>();

    public MsgDBRespuesta(ArrayList<String> to, String from, String fecha) {
        super(to, from, fecha, "MsgDBRespuesta", true);
    }

    public String getCodigoPeticion() {
        return CodigoPeticion;
    }

    public void setCodigoPeticion(String CodigoPeticion) {
        this.CodigoPeticion = CodigoPeticion;
    }
}

File: SistemaTrackingEscritorio/SistemaTrackingBiblioteca/Mensajes/MsgDBRespuesta.cs
using SistemaTrackingBiblioteca.Entidades;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SistemaTrackingBiblioteca.Mensajes
{
    public class MsgDBRespuesta : Mensaje
    {
        public MsgDBRespuesta()
        {
            base.Tipo = this.GetType().Name;
        }

        /// <summary>
        /// Codigos:
        /// <para>- "Login" = return: Cuenta</para>
        /// <para>- "CrearCuenta" = return: Cuenta</para>
        /// <para>- "BorrarCuenta" = return: void</para>
        /// <para>- "CrearGrupo" = return: Grupo</para>
        /// <para>- "GetGrupo" = return: Grupo</para>
        /// <para>- "GetGrupoPorAnfitrion" = return: Grupo(s)</para>
        /// <para>- "GetGrupoPorIntegrante" = return: Grupo(s)</para>
        /// <para>- "AgregarCuentaAGrupo" = return: Grupo</para>
        /// <para>- "BorrarCuentaDeGrupo" = return: Grupo</para>
        /// <para>- "BorrarGrupo" = return: void</para>
        /// <para>- "GetHistorialPorGrupo" = return: Historial(s)</para>
        /// <para>- "GetHistorialPorCuenta" = return: Historial(s)</para>
        /// <para>- "GetCuentas" = return: Cuenta(s)</para>
        /// </summary>
        public string CodigoPeticion { get; set; }

        public List<Cuenta> ReturnCuenta = new List<Cuenta>();

        public List<Grupo> ReturnGrupo = new List<Grupo>();

        public List<Historial> ReturnHistorial = new List<Historial>();
    }
}
